.github/workflows/release.yml

```diff
@@ -124,6 +124,31 @@ jobs:
           name: enarx-x86_64-windows-msi
           path: target\wix\enarx-x86_64-windows.msi
 
+  sign-x86_64:
+    needs: [ build-x86_64 ]
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/download-artifact@v3
+      with:
+        name: enarx-x86_64-unknown-linux-musl
+    - name: Generate the enarx keep signature
+      env:
+        SEV_ID_KEY: ${{ secrets.SEV_ID_KEY }}
```

Comment:
The problem here (and why CI is failing) is that these secrets aren't available for a PR created from another repository, which they also shouldn't be (since that would allow any contributor to get stuff signed by us).
Perhaps we should check whether these secrets are unavailable, and if they are (like they will be in e.g. PR initiated CI), generate and use a new key during this process?

Reply:
The secrets are not available when the workflow is executed from a fork, so I put in a workaround where new keys would be generated for that run. Situation should be addressed.